storio-sqlite-annotations/src/main/java/com/pushtorefresh/storio/sqlite/annotations/StorIOSQLiteType.java

```diff
@@ -14,9 +14,34 @@
 public @interface StorIOSQLiteType {
 
     /**
-     * Required: Specifies table name
+     * Optional: Specifies table name for all operations
+     * Should be setted as default if there is no name for some particular operation
      *
      * @return table name
+     * @see #insertTable()
+     * @see #updateTable()
+     * @see #deleteTable()
      */
-    String table();
+    String table() default "";
+
+    /**
+     * Optional: Specifies table name for insert operation
+     *
+     * @return table name
+     */
+    String insertTable() default "";
```

Comment:
But I see only discussion about ContentResolver. I mean, you think it will be useful to provide ability to do `inserts/updates/deletes` to/from different tables? I don't see much value in this :)